DataManagementSystem/Agent/FTSAgent.py

```diff
@@ -664,6 +676,8 @@ def processRequest( self, request ):
 
     except escapeTry:
       # This clause is raised when one wants to return from within the try: clause
+      # only put back jobs that were monitored
+      ftsJobs = jobsToMonitor
       pass
```

Comment:
This `pass` becomes useless then.

Also (very minor...): use 
```
except Exception as exceptMessage:
```
instead of 
```
except Exception, exceptMessage:
```